File: problems/c/conundrum/conundrum.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    cipher_text = input()
    cipher_size = len(cipher_text)
    per_times = cipher_size // 3
    new_text = "PER" * per_times
    print(sum([1 for i in range(cipher_size) if cipher_text[i] != new_text[i]]))


if __name__ == '__main__':
    main()

File: problems/c/conundrum/conundrum.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    string cipher;
    string name = "PER";
    int number_days = 0;

    cin >> cipher;

    for (size_t i=0; i<cipher.size(); i++) {
        if (name[i % 3] != cipher[i]) {
            number_days++;
        }
    }

    cout << number_days << endl;
}